### PR TITLE
Make the governance policy for this repo clear

### PR DESCRIPTION
**Pull request by KengoTODA**
To solve problems caused by our unclear governance policy for this project, create a `GOVERNANCE.md` at the root directory.
I referred to [The nodejs governance policy](https://github.com/nodejs/node/blob/c93a639788145d45f96e788ab169cd3d8a5e251e/GOVERNANCE.md) and the [Minimum Viable Governance from GitHub](https://github.com/github/MVG/tree/3e2371415353a96097974e32407c2af097004b8d/), but the content is made from scratch to enable our small start.

After we merge this change, I will configure the branch protection rule to change the number of necessary approval from 1 to 2. Other automations such as [stale action](https://github.com/actions/stale) and auto-merge will come next.

### CODE REVIEW

**Comment by iloveeclipse** [comment]
What is the reason to request two approvals for a PR?

File: GOVERNANCE.md
Comment (ThrawnCA) [inline GOVERNANCE.md:15]: "got" -> "gets"

"the same to" -> "the same as the"

"authors of" -> "author(s) of"

Perhaps also add something about how to handle pull requests where all or most of the team has contributed as authors?

**Review by ThrawnCA — CHANGES_REQUESTED** [review]

File: GOVERNANCE.md
Comment (KengoTODA) [reply GOVERNANCE.md]: > Perhaps also add something about how to handle pull requests where all or most of the team has contributed as authors?

Any idea do you have? How about asking project lead (iloveeclipse) to make a judge?

**Comment by KengoTODA** [comment]
> What is the reason to request two approvals for a PR?

It is mostly the same to the unwritten rule in this repo. Most of our merged PRs have two approvers, or one approver and another merger.

I personally think that it's a good habit to keep the project secure. We can assume that at least multiple people made judgement on its implementation and maintainability. We can also avoid the situation like "one strong committer" drives everything so no decision is open nor understandable.

**Review by ThrawnCA — APPROVED** [review]

File: GOVERNANCE.md
Comment (ThrawnCA) [reply GOVERNANCE.md]: One option: it's okay to have approvers who are also authors if there are at least three approvals (after the most recent changes). So everyone's work has been reviewed by at least two people who didn't write it, even if they wrote other parts.

File: GOVERNANCE.md
Comment (iloveeclipse) [inline GOVERNANCE.md:15]: > Approver should not be the same as the author(s) of the change.

This is contradictory to the current state / your comment: 

> It is mostly the same to the unwritten rule in this repo. Most of our merged PRs have two approvers, or one approver and another merger.

Currently if we have author from committers, we only want one more approval.
With this proposal, we will need two approvals - one more as today.

**Review by iloveeclipse — CHANGES_REQUESTED** [review]

File: GOVERNANCE.md
Comment (KengoTODA) [reply GOVERNANCE.md:15]: Yes, this is a challenge as noted at https://github.com/orgs/spotbugs/teams/everyone/discussions/9?from_comment=4#discussion-9-comment-4

I think it's reasonable change for governance, how do you think?

Comment (iloveeclipse) [reply GOVERNANCE.md:15]: I don't think so. One can't force people to do this if they don't want or can't. I see mostly you and ThrawnCA active, and while I'm still there, I simply have no time to spent on reviews, especially if they are for features I'm not interested personally (lile gradle support etc). So by forcing two additional reviews you force me to do something I won't, and as a result I will be unhappy and feel me bad because on the one side I know your work waits for my review but on the other side I can't afford to spend reasonable amount of time for this (or don't want to review this).

Comment (KengoTODA) [reply GOVERNANCE.md:15]: OK I understood. Thanks. I will remove the limitation later.

File: GOVERNANCE.md
Comment (iloveeclipse) [inline GOVERNANCE.md:5]: Term -> Team

File: GOVERNANCE.md
Comment (iloveeclipse) [inline GOVERNANCE.md:7]: maintain -> maintains

File: GOVERNANCE.md
Comment (iloveeclipse) [inline GOVERNANCE.md:15]: will -> can

File: GOVERNANCE.md
Comment (iloveeclipse) [inline GOVERNANCE.md:17]: will -> can

File: GOVERNANCE.md
Comment (iloveeclipse) [inline GOVERNANCE.md:21]: user -> users ?
issue -> issues?
to know -> to get

File: GOVERNANCE.md
Comment (iloveeclipse) [inline GOVERNANCE.md:23]: has no -> is not
it is -> it was
will -> can
closed -> close

File: GOVERNANCE.md
Comment (iloveeclipse) [inline GOVERNANCE.md:29]: to nominate -> for nomination

**Review by iloveeclipse — CHANGES_REQUESTED** [review]
I think I'm OK in general, few style/language issues.

**Comment by KengoTODA** [comment]
Thanks @iloveeclipse, I've reflected your review to the doc. It helped me a lot to make it better! :)

**Review by iloveeclipse — APPROVED** [review]

**Pull request merged**
